swagger.yaml

```diff
@@ -864,4 +897,29 @@ components:
                 enum:
                   - HEAD
                   - TAIL
-
+    SecretSanta:
+      type: object
+      properties:
+        langauge:
+          type: string
+          enum:
+            - es
+            - en
+        participants:
+          type: array
+          items:
+            maxLength: 500
+            minLength: 3
+            type: object
+              properties:
```

Comment:
We don't have exclusions here but I see they are mentioned somewhere else in the PR

Reply:
Yeah, I did not added in the first iteration to unblock you ASAP.

Reply:
Exclusions are ready now.